.github/workflows/format.yml

```diff
@@ -0,0 +1,31 @@
+name: Format
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+defaults:
+  run:
+    working-directory: javascript
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14.x"
+      - name: Format
+        run: |
+          npm ci
+          npm run prettier
```

Comment:
I also vote that this should be a check, without auto-commit. Specially since I am not sure if `github.head_ref` can point to something other than PR (e.g. master)

Reply:
@msridhar Done in https://github.com/uber/piranha/pull/84/commits/443a91e133573f9916458ed35c2885914dad8f9a Auto-fixing code was a bad idea as it pushes to master so changed `--write` to `--check`.